projects/igniteui-angular/src/lib/switch/switch.component.ts

```diff
@@ -368,6 +370,7 @@ export class IgxSwitchRequiredDirective extends CheckboxRequiredValidator { }
 
 /**
  * @hidden
+ * @internal
```

Comment:
According to the newest version of the guidelines Modules can't be `@internal`

Reply:
Thanks for the remark, it's done now.